spec/component/importers/activity_insight_importer_spec.rb

```diff
@@ -4993,6 +4993,27 @@
                                       publication: existing_pub) }
       let!(:existing_pub) { create(:publication) }
 
+      context 'when the publication to be imported does not have a file' do
+        let!(:existing_import2) { create(:publication_import,
+                                         source: 'Activity Insight',
+                                         source_identifier: '190706413568',
+                                         publication: pub1) }
+        let!(:aif1) { create(:activity_insight_oa_file,
+                             publication: pub1, version: 'publishedVersion',
+                             file_download_location: fixture_file_open('test_file.pdf'),
+                             intellcont_id: '190706413568') }
+        let!(:pub1) { create(:publication) }
+
+        let!(:file_download_directory1) { aif1.file_download_location.model_object_dir }
+
+        it 'deletes the ActivityInsightOAFile and downloaded file' do
```

Comment:
I think we should test and describe the two potential outcomes here in their own contexts.  At least so the behavior is documented.  

- if an ActivityInsightOAFile exists: it gets deleted
- if an ActivityInsightOAFile does not exist: nothing happens